modules/monitoring-configuring-the-cluster-monitoring-stack.adoc

```diff
@@ -50,7 +50,8 @@ data:
     *prometheusK8s*:
       *volumeClaimTemplate:
        spec:
-         storageClassName: gluster-block
+         storageClassName: fast
+         volumeMode: filesystem
```

Comment:
Should be
volumeMode: Filesystem
if we use filesystem, PVCs can not be created,
# oc -n openshift-monitoring describe sts alertmanager-main
Events:
  Type     Reason        Age                         From                    Message
  ----     ------        ----                        ----                    -------
  Warning  FailedCreate  3m48s (x12 over 3m58s)      statefulset-controller  create Pod prometheus-k8s-0 in StatefulSet prometheus-k8s failed error: failed to create PVC prometheus-k8s-db-prometheus-k8s-0: PersistentVolumeClaim "prometheus-k8s-db-prometheus-k8s-0" is invalid: spec.volumeMode: Unsupported value: "filesystem": supported values: "Block", "Filesystem"

Reply:
Thanks @juzhao, fixing in https://github.com/openshift/openshift-docs/pull/19573.